email_app/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 ### Changed
 - Update to Node JS to ^8.0.0
 - Update @maidsafe/safe-node-app to v0.6.1 
```

Comment:
We don't need to keep this here as the changelog is to show the changes compared with the previous version released, in this case we've upgraded the dependency twice but compared to WHM v0.4.4 we just moved from v0.6.0 to v0.7.0.